Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -55,6 +57,14 @@ public float FlatHandThreshold
             set { flatHandThreshold = value; }
         }
 
+        [SerializeField]
+        [Tooltip("With this active, solver will follow hand rotation until the menu is sufficiently aligned with the gaze, at which point it faces the camera.")]
+        private bool followHandUntilFacingCamera = false;
```

Comment:
It would be good to make these two fields configurable at runtime, [following the principle 'support configuring in editor and at runtime](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#support-configuring-components-both-in-editor-and-at-run-time)

Reply:
That makes sense! I can expose them as public properties.